iotdb/src/test/java/org/apache/iotdb/db/postback/sender/FileManagerTest.java

```diff
@@ -229,46 +228,46 @@ public void testGetNowLocalFileList() throws IOException {
         if (!file.getParentFile().exists()) {
           file.getParentFile().mkdirs();
         }
-        if (!file.exists()) {
-          file.createNewFile();
+        if (!file.exists() && !file.createNewFile()) {
+          LOGGER.error("Can not create new file {}", file.getAbsoluteFile());
         }
       }
     }
-    manager.setNowLocalFiles(new HashMap<>());
-    manager.getNowLocalFileList(new String[]{SENDER_FILE_PATH_TEST});
-    fileList = manager.getNowLocalFiles();
+    manager.setCurrentLocalFiles(new HashMap<>());
+    manager.getCurrentLocalFileList(new String[]{SENDER_FILE_PATH_TEST});
+    fileList = manager.getCurrentLocalFiles();
     assert (allFileList.size() == fileList.size());
     for (Entry<String, Set<String>> entry : fileList.entrySet()) {
       assert (allFileList.containsKey(entry.getKey()));
-      System.out.println("allFileList");
+      LOGGER.info("allFileList");
```

Comment:
how about change these to Logger.debug() level?

Reply:
good advice